### PR TITLE
Added Pods with Mars

### PR DESCRIPTION
**Pull request by robertoduessmann**
Added PODS with MARS 160g (Australia, 06.10.17).

For: https://github.com/ProductCrawler/product_crawler/issues/61
refs http://www.marschocolate.com.au/products/pods/pods-with-mars/

### CODE REVIEW

File: PODS with MARS 160g (Australia, 06.10.17)/pods_with_mars.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Product_Name; PODS with MARS (160g)
```
Comment (Justicimo) [inline PODS with MARS 160g (Australia, 06.10.17)/pods_with_mars.txt:1]: you should change the (160g) -> (100g) to match the data below. Would be more explicit.

**Comment by Justicimo** [comment]
Can you please update the .png to show the nutritional facts rather than packaging and update .txt's serving size to match the image (100g)  then create a new pull request.

File: PODS with MARS 160g (Australia, 06.10.17)/pods_with_mars.txt (hunk 1 above)
Comment (KAUTH) [reply PODS with MARS 160g (Australia, 06.10.17)/pods_with_mars.txt:1]: @Justicimo 160g is the correct input. We are talking about the product's actual weight

**Comment by KAUTH** [comment]
@robertoduessmann @Justicimo is right about the PNG file, please provide a screenshot of the product's label

**Comment by robertoduessmann** [comment]
Thanks for feedback @Justicimo and @KAUTH .
I was having trouble to found a better image, but now I found it.
I've created pull request https://github.com/ProductCrawler/product_crawler/pull/75 to update image to show the nutritional facts.

**Comment by KAUTH** [comment]
@robertoduessmann don't worry about it, I had already added a new image. Thanks for the contribution!!

**Pull request merged**